src/index.ts

```diff
@@ -23,12 +23,13 @@ export = function writableDOM(
   let scanNode: Node | null = null;
   let resolve: void | (() => void);
   let isBlocked = false;
+  let inlineHostNode: Node | null = null;
 
   return {
     write(chunk: string) {
       doc.write(chunk);
 
-      if (pendingText) {
+      if (pendingText && isInlineHost(pendingText.parentElement!)) {
```

Comment:
could you just check if `inlineHostNode != null`?

Reply:
yes, I've applied the change, sorry, silly mistake 😅, thank you very much! 😄